sklearn/metrics/tests/test_ranking.py

```diff
@@ -511,6 +509,29 @@ def test_precision_recall_curve():
     assert_equal(p.size, t.size + 1)
 
 
+def test_precision_recall_curve_nan():
+    y_true, _, probas_pred = make_prediction(binary=True)
+    _test_precision_recall_curve(y_true, probas_pred)
+
+    # Use {-1, 1} for labels; make sure original labels aren't modified
+    y_true[np.where(y_true == 0)] = -1
+    y_true_copy = y_true.copy()
+    _test_precision_recall_curve(y_true, probas_pred)
+    assert_array_equal(y_true_copy, y_true)
+
+    labels = [0, 0, 0, 0]
+    predict_probas = [1, 2, 3, 4]
```

Comment:
please call this decision instead, because having proba > 1 doesn't correspond to a natural use case.